opentracing/tracer.py

```diff
@@ -40,31 +41,80 @@ class Tracer(object):
     def __init__(self):
         self._noop_span_context = SpanContext()
         self._noop_span = Span(tracer=self, context=self._noop_span_context)
+        self._active_span_source = NoopActiveSpanSource()
 
-    def start_span(self,
-                   operation_name=None,
-                   child_of=None,
-                   references=None,
-                   tags=None,
-                   start_time=None):
+    @property
+    def active_span_source(self):
+        return self._active_span_source
+
+    def start_active_span(self, operation_name, tags=None, start_time=None):
```

Comment:
I'd say to keep them short we can:
```
tracer.start_active()
tracer.start_manual()
```
mostly because the Java API introduces the concept of `active` and `manual` operation modes